go.mod

```diff
@@ -28,7 +28,9 @@ require github.com/golang/mock v1.6.0
 require (
 	github.com/ipfs/go-cid v0.1.0
 	github.com/multiformats/go-multiaddr v0.5.0
+	github.com/spf13/cobra v1.5.0
```

Comment:
Demos in `samples` use `kingpin`.
I don't have a preference, but we should probably aim at being consistent.

Reply:
I considered both packages and `cobra` looked more popular. I used `cobra` before and it was easy to deal with.